Dockerfile.production

```
@@ -0,0 +1,5 @@
+ARG ROS_VERSION=foxy
+FROM aica-technology/modulo/development:${ROS_VERSION} as remote-development
+
+# build modulo
+RUN /bin/bash -c "source /opt/ros/$ROS_DISTRO/setup.bash; colcon build"
```

I personally prefer doing that instead of changing the user to ${USER} on line 10 in the development Dockerfile
```suggestion
RUN su ${USER} -c /bin/bash -c "source /opt/ros/$ROS_DISTRO/setup.bash; colcon build"
```